rpc_util_test.go

```diff
@@ -290,3 +292,130 @@ func BenchmarkGZIPCompressor512KiB(b *testing.B) {
 func BenchmarkGZIPCompressor1MiB(b *testing.B) {
 	bmCompressor(b, 1024*1024, NewGZIPCompressor())
 }
+
+type mockCompressor struct {
+	DecompressedData []byte
```

Comment:
why are these attributes upper case? We are not exporting them

Reply:
changed to lowercase